### PR TITLE
swiftformat 0.17 (new formula)

### PR DESCRIPTION
**Pull request by nicklockwood**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/swiftformat.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Swiftformat < Formula
+  desc "Formatting tool for Swift source code"
+  homepage "https://github.com/nicklockwood/SwiftFormat"
+  url "https://github.com/nicklockwood/SwiftFormat/archive/0.17.tar.gz"
+  sha256 "68cfc0f783ca331a1fda1c8b5c5a670b6c42be72a44a34b519d61d308b2fe8ea"
+
+  def install
+    bin.install "CommandLineTool/swiftformat"
```
Comment (bfontaine) [inline Formula/swiftformat.rb]: We need to build from source here.

File: Formula/swiftformat.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  end
+
+  test do
+    system "#{bin}/swiftformat"
```
Comment (bfontaine) [inline Formula/swiftformat.rb]: Please write a test that does something more substantial than e.g. printing the help. You can write a test Swift file and call `swiftformat` on it.

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Comment by dunn** [comment]
Ping re @bfontaine's comments

**Comment by woodruffw** [comment]
Seems to have been superseded by #7138.

**Pull request closed without merging**